CMakeLists.txt

```diff
@@ -1,25 +1,28 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.11)
```

Comment:
Shouldn't we require 3.14? I'm looking at this guide https://cliutils.gitlab.io/modern-cmake/chapters/projects/fetch.html

Reply:
Indeed, I updated the minimum required version.